src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +54,93 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철 역 목록을 조회하면 모든 지하철 역 정보를 응답받습니다.")
+    @Test
+    void showStations() {
+        // given
+        Map<String, String> body1 = makeStationRequestBody("station1");
+        Map<String, String> body2 = makeStationRequestBody("station2");
+
+        RestAssured
+                .given()
+                .log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .body(body1)
+                .when()
+                .post("/stations");
+
+        RestAssured
+                .given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .body(body2)
+                .when()
+                .post("/stations");
```

Comment:
지금 인수테스트들을 보면 몇몇 동작(지하철역 생성, 조회등)들이 반복되고 있어요. 이를 테스트마다 새로 작성해준다면, 테스트 블럭의 길이도 너무 길어질 수 있고, 만약 이 스펙이 변경되었을 경우 모든 테스트블럭의 로직을 바꿔줘야하는 문제가 같이 생기죠. 그렇기에 이런 부분들은 공통으로 처리할 수 있어도 좋을 것 같네요! 

StationSteps같은 공통 유틸 클래스가 하나 나와도 좋을 것 같네요!

그리고 body1, body2라는 이름만봐서는 뭔지 이해하기 힘들 것 같은 이름인데 좀 더 명확한 이름을 짓는건 어떨까요? 

```suggestion
        stationsSteps.지하철역_생성(강남역);
        stationsSteps.지하철역_생성(신분당선);
```

이런식으로도 가능하지 않을까요!?